packages/ui/src/components/va-toast/VaToast.vue

```diff
@@ -82,6 +86,7 @@ export default defineComponent({
     },
     render: { type: Function },
     ariaCloseLabel: { type: String, default: '$t:close' },
+    role: { type: StringWithAutocomplete<'alert' | 'alertdialog' | 'status'> },
```

Comment:
Here vscode emits: `'StringWithAutocomplete' only refers to a type, but is being used as a value here.

Reply:
Thank you very much, @m0ksem.

I have tried to add it as a prop, but I have meeting the error with message I comment about. How can I deal with that?

Reply:
When you type `prop: { type: Something }` something must be valid JS class, like Number, String, etc. You can add more specific type with TS using [PropType](https://vuejs.org/api/utility-types.html#proptype-t).

```suggestion
    role: { type: String as PropType<StringWithAutocomplete<'alert' | 'alertdialog' | 'status'>> },
```

Role still can be any String on runtime in browser, but in IDE there will be suggestions that role can be one of three you proposed.